src/components/collapse/collapse.js

```diff
@@ -187,6 +187,9 @@ export default {
     }
     this.emitState()
   },
+  updated () {
```

Comment:
This actually runs when the component's vdom has updated, and doesn't monitor for changes outside of the component.

What might be a better approach to this, is to update the `v-b-toggle` directive to use the `observeDom` utility to monitor changes to the trigger element's attributes (shallow observer).

I will look into this.

Reply:
Although just testing in the playground, this appears to work for most cases.